nominal-streaming/src/stream.rs

```diff
@@ -127,6 +131,16 @@ impl NominalDatasetStreamBuilder {
         self
     }
 
+    #[cfg(feature = "logging")]
+    pub fn enable_logging(self) -> Self {
+        self.init_logging(None)
+    }
+
+    #[cfg(feature = "logging")]
+    pub fn enable_logging_with_directive(self, log_directive: &str) -> Self {
```

Comment:
is this for Rust users? the enable logging thing is really just for me/you to debug, so changing the directive can just be via `RUST_LOG=info` or w/e (but in practice only debug logging is actually useful if things aren't working, hence didn't expose it more widely).